modules/library/jdbc/src/main/java/org/geotools/data/jdbc/FilterToSQL.java

```diff
@@ -824,6 +824,16 @@ protected void visitBinaryComparisonOperator(BinaryComparisonOperator filter, Ob
         Class rightContext = getExpressionType(left);
         Class leftContext = getExpressionType(right);
 
+        encodeBinaryComparisonOperator(filter, extraData, left, right, leftContext, rightContext);
+    }
+
+    protected void encodeBinaryComparisonOperator(
```

Comment:
This class is extended for all databases, please add javadoc for this method so that subclassers know what it is for.

Reply:
fixed